docs/modules/general/pages/architecture-overview/remoteComponents.adoc

```diff
@@ -23,19 +23,36 @@ OneCX Slot is a standard component used in the html content templates. It uses c
 Whenver it is desired to display Remote Component, a Slot is needed to act as a host element for it. For OneCX Core Applications, there are existing Slots, which can be used to configure displayed content (especially Shell Application's slots).
```

Comment:
**Whenever** it is desired to display **a** Remote Component,  ...
... especially Shell Application's **Slots**